backend/inspirehep/records/api/literature.py

```diff
@@ -720,7 +721,12 @@ def import_arxiv(arxiv_id):
             f"An error occurred while parsing article oai:arXiv.org:{arxiv_id}."
         ) from exc
 
-
+@backoff.on_exception(
+        backoff.constant,
+        (ImportTimeoutError),
+        max_tries=1,
+        interval=10,
```

Comment:
this will make the users to wait 10 seconds, we should use `max_time=10` instead